coderadar-graph/src/main/java/io/reflectoring/coderadar/graph/analyzer/adapter/ResetAnalysisAdapter.java

```diff
@@ -10,22 +9,17 @@
 public class ResetAnalysisAdapter implements ResetAnalysisPort {
   private final CommitRepository commitRepository;
   private final ProjectRepository projectRepository;
-  private final AnalyzingJobRepository analyzingJobRepository;
 
   public ResetAnalysisAdapter(
-      CommitRepository commitRepository,
-      ProjectRepository projectRepository,
-      AnalyzingJobRepository analyzingJobRepository) {
+      CommitRepository commitRepository, ProjectRepository projectRepository) {
     this.commitRepository = commitRepository;
     this.projectRepository = projectRepository;
-    this.analyzingJobRepository = analyzingJobRepository;
   }
 
   @Override
   public void resetAnalysis(Long projectId) {
     commitRepository.resetAnalyzedStatus(projectId);
     while (projectRepository.deleteProjectFindings(projectId) > 0) ;
     while (projectRepository.deleteProjectMetrics(projectId) > 0) ;
```

Comment:
I get the idea, but for readability reasons we should either rewrite that code or write some comments, right?

Reply:
I've added comments to the repository methods and the loops.